packages/framework/fluid-static/package.json

```diff
@@ -65,8 +65,8 @@
 		"test": "npm run test:mocha",
 		"test:coverage": "c8 npm test",
 		"test:mocha": "npm run test:mocha:esm && echo skipping cjs to avoid overhead - npm run test:mocha:cjs",
-		"test:mocha:cjs": "mocha --recursive \"dist/test/**/*.spec.*js\" --exit -r node_modules/@fluid-internal/mocha-test-setup",
-		"test:mocha:esm": "mocha --recursive \"lib/test/**/*.spec.*js\" --exit -r node_modules/@fluid-internal/mocha-test-setup",
+		"test:mocha:cjs": "mocha --recursive \"dist/test/**/*.spec.*js\" --exit",
```

Comment:
Most of our packages use mocha-test-setup this way, though I'm always fuzzy on what it's doing - what's the effect of removing it here (and would we do the same for other packages)?

Reply:
Most projects use a '.mocharc.cjs' that already imports the mocha-test-setup:
```ts
const getFluidTestMochaConfig = require("@fluid-internal/mocha-test-setup/mocharc-common");
```